src/pages/experiments/[experimentId]/data-exploration/components/generic-gene-table/ComponentActions.jsx

```diff
@@ -31,7 +32,7 @@ const ComponentActions = (props) => {
       newGenes = displayedGenes.filter((gene) => !selectedGenes.includes(gene));
     }
 
-    dispatch(loadGeneExpression(experimentId, newGenes, componentType));
+    dispatch(loadGeneExpression(experimentId, newGenes, componentType, expressionType));
```

Comment:
instead of having to make any changes to `ComponentActions` and make it "know" and "care" about `expressionType`, what you can do is to read the value of `expressionType` directly from `loadGeneExpression` action, similarly to how it is done in action `loadPaginatedGeneProperties`, line 16 as an example.

Reply:
Oh ya, that would make it better API-wise for the function too. So better change it?